README.md

```diff
@@ -1,242 +1,82 @@
-# Gradle Semver Plugin
-
-This plugin adds a flexible approach to adding semantic versioning to your gradle project using git history.
-It supports multi-module gradle projects, running in CI, and multiple types of git strategies.
-
-- [Usage](#usage)
-- [Overview](#overview)
-- [Using with CI](#using-with-ci)
-- [Version Calculation](#version-calculation)
-- [Branch Matching Strategy](#branch-matching-strategy)
-- [Unsupported](#not-supported)
-
-## Usage
+# Semver Gradle Plugin
+
+This Semver Gradle plugin provides a simple approach to
+adding semantic versioning to your gradle project using git
+history regardless of git strategies.
+
+At a glance, this plugin provides the following features:
+
+- Support for stages (`rc`, `beta`, `stable`, `snapshot`, etc. - see below)
+- Support for modifiers (`auto`, `patch`, `minor`, `major`)
+- Support for branch-based version calculations
+- Support for overriding the version
+- Support for setting an alternate initial version
+- Support for specifying alternate main and development branch names
+- Support for appending build metadata (format: `+<yyyyMMddHHmmss>`)
```

Comment:
`Support for ...` here seems repetitious to me... Please consider moving it to the above line. Maybe something like this: `At a glance, this plugin supports the following features:` and then just list the items. 🤷🏻‍♂️